sink_test.go

```diff
@@ -87,11 +98,10 @@ func TestRegisterSinkErrors(t *testing.T) {
 		{"http*", "not a valid scheme"},
 	}
 
+	r := newSinkRegistry()
```

Comment:
There does not appear to be any shared state between individual test cases.
Should this be defined inside the subtest?